### ISSUE TITLE
Not able to retrieve the latest version of the theme as a Hugo module

### ISSUE THREAD

**Issue by igor-baiborodine** [Lednerb/bilberry-hugo-theme#344]
Currently, the latest release is at v2.13.0. But when trying to fetch the theme as a Hugo module, the only available version is v2.10.0, which is the last version released before migration to Hugo modules.

```shell
$ hugo mod clean --all
$ hugo mod get -u github.com/Lednerb/bilberry-hugo-theme
```
![image](https://user-images.githubusercontent.com/1027701/152993384-c7e9f15d-0aea-4934-bf86-f11f40b48d32.png)

![image](https://user-images.githubusercontent.com/1027701/152994000-d330f239-9fe4-4c2d-bce5-d9ea7f7e5291.png)

**Comment by igor-baiborodine** [comment]
It looks like the major version was supposed to be bumped to v3 when the Hugo module was introduced.
For more details, see this doc: https://github.com/golang/go/wiki/Modules#releasing-modules-v2-or-higher.

**Comment by Lednerb** [comment]
I also think there is no problem with switching to v3, you can bump the major version.

**Comment by igor-baiborodine** [comment]
Fixed with commit [7eb2b347](https://github.com/Lednerb/bilberry-hugo-theme/commit/7eb2b347f82fc83361cfa18eb1b3424aabd3fabd) and bumping the major version to `v3`:
![image](https://user-images.githubusercontent.com/1027701/154266280-c6f7ae20-64a2-4192-90fe-0632a047f916.png)
